app/Model/SubtaskTimeTrackingModel.php

```diff
@@ -87,6 +87,8 @@ public function getTaskQuery($task_id)
                         self::TABLE.'.start',
                         self::TABLE.'.time_spent',
                         self::TABLE.'.user_id',
+                        self::TABLE.'.is_billable',
```

Comment:
these are fields from a different plugin that have been slipped in. i have removed them.